src/main/java/org/folio/circulation/domain/ItemLostRequiringCostsEntry.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.circulation.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.With;
+
+@AllArgsConstructor
+@Getter
+@With
+public class ItemLostRequiringCostsEntry {
```

Comment:
I don't think that `entry` adds much to the name of this, the space could be used to more specifically describe the type

```suggestion 
public class LostItemRequiringActualCostsFee {
```